proto/viam/app/v1/app.proto

```diff
@@ -877,6 +877,9 @@ message Module {
   string module_id = 1;
   // The id of the organization that owns the module
   string organization_id = 10;
+  // The public namespace of the organization that owns the module
+  // This is empty if no public namespace is set
+  string public_namespace = 12;
```

Comment:
nit: maybe add this so the numbering is correctly in order ?

Reply:
Add this so the numbering is in the correct order? There is already an `=11;` 

Do you just mean sort the entries in the message by the id?

Reply:
yeah just sorted in ID so its clear what the next value needs to be

Reply:
Sure -- I just left because it was already unsorted but happy to re-sort